### DISCUSSION TITLE
COMFAST CF-727B (RTL8822BU) confirmed working

### DISCUSSION THREAD

**Discussion by nomorecoolnicknames** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#176] [Show and tell]
WIth default Chinese drivers Wi-Fi driver dies on every reboot, your one looks better and works as it should be. Thanks! But of course with no Bluetooth, as it is impossible I think.

**Comment by chris1111** [comment]
> WIth default Chinese drivers Wi-Fi driver dies on every reboot, your one looks better and works as it should be. Thanks! But of course with no Bluetooth, as it is impossible I think.


USB WIFI Adaptor never working for bluetooth device in macOS you need to buy a compatible macOS USB bluetooth
